### ISSUE TITLE
Mac menu bar icon

### ISSUE THREAD

**Issue by bonebox** [beville/ComicStreamer#12]
A couple things I noticed on my Macbook Pro (retina):
1. When I hover over the menu bar icon, I get a spinner and cannot click on the icon.
2. More of a quibble, but the icon is not retina.

**Comment by beville** [comment]
Funny you should post right this now.  My current Mac development machine is an older 32-bit  Mac mini, running Snow Leopard. However, last night I had the chance to play on a newer mac running Mavericks, and ran into the non-functional menu bar problem.  

Unfortunately I won't be able to fix this until I have a 64-bit dev machine.  My plan is to upgrade to a new mini once Apple announces the next refresh, but I've been waiting over 6 months now!  Fortunately, the only two items in the menu are "Quit" and "Open WebUI"

The icon is meant to be temporary, but could you share a link that explains retina format?  Again, my mac context is stuck in 2009 right now :-)

**Comment by bonebox** [comment]
Yeah, the Mac Mini could use a little love.  I'm betting they release a new version later this year.  Here's some info on Mac development for retina displays.  The 2nd link is probably the most relevant to what you need.

https://developer.apple.com/library/mac/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/Introduction/Introduction.html

https://developer.apple.com/library/mac/documentation/GraphicsAnimation/Conceptual/HighResolutionOSX/Optimizing/Optimizing.html#//apple_ref/doc/uid/TP40012302-CH7-SW1

**Comment by bonebox** [comment]
It may just be a matter of adding another image to the project double the size of the original and naming it the same as the original except with `@2x` appended.  So if the original is `comic_streamer.png` the retina image would be `comic_streamer@2x.png`.

**Comment by mike-ferenduros** [comment]
The menubar thing looks to be a known issue on 10.9; replacing lib/rumps.py with the version from this fork fixes it for me:
https://github.com/tito/rumps
